### DISCUSSION TITLE
Can I share a high-level solution based on Netflix-DGS here?

### DISCUSSION THREAD

**Discussion by babyfish-ct** [Netflix/dgs-framework#1020] [Ideas]
Can I share a high-level solution based on Netflix-DGS here?

I built a high-level framework based on Netflix-DGS. let user quickly build a GraphQL service on RDBMS (using kotlin and R2DBC)

I don't know will doing this offends everyone, can I share the project link and youtube video link here?

Anyway, thanks a lot to Netflix-DGS, it bridges graphql-java and SpringBoot nicely and saves me a lot of work.

**Comment by paulbakker** [comment]
Yes, of course, I'm sure that will be useful to others!

As for the actual code of the framework - it would probably be best to have that available in a separate github repo instead of adding it to the main DGS repository. We do the same with all the Netflix specific additions we built on top of the framework for example. That way the release cycles aren't tied up to each other.

If you have a youtube video etc. to link to, we could add a "community examples" section to the website maybe to make it easy to find?

**Reply by babyfish-ct** [reply]
"If you have a youtube video etc. to link to, we could add a "community examples" section to the website maybe to make it easy to find?"

Sorry, I didn't notice this just now. Sure, this is very good.

**Comment by babyfish-ct** [comment]
@paulbakker Thanks

1. project link: 
    [https://github.com/babyfish-ct/graphql-provider](https://github.com/babyfish-ct/graphql-provider)
2. youtube video link:
    [https://www.youtube.com/watch?v=ky6HQQwo-3w](https://www.youtube.com/watch?v=ky6HQQwo-3w)
